sql/rewards/rewards.go

```diff
@@ -54,3 +54,22 @@ func List(db sql.Executor, coinbase types.Address) (rst []*types.Reward, err err
 		})
 	return
 }
+
+// RewardPerLayer returns the reward per layer from `rewards` table.
+func RewardPerLayer(db sql.Executor, layerID uint32) (rwd *types.Reward, err error) {
```

Comment:
please don't use uint32 for layer id in the code, the type should be consistent with the rest of the codebase

Reply:
> there are on average 50 rewards per layer for different coinbases. all of them may have different weight behind them. > which one do you want to get?

but node will get reward only for block, which will be constructed after `Hair`. It can be several blocks in hair output, but is should be extraordinary situations, and our estimation will not work in that cases

Reply:
based on your messages i thought that you are going to provide an estimate for this layer based on the previous layers results, but this is not what is going on in this change